packages/client/ui/react-ui/src/components/common/InputOTP.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+
+import * as React from "react";
+import { OTPInput, OTPInputContext } from "input-otp";
+import { createContext, useContext } from "react";
+import { classNames } from "@/utils/classNames";
+
+// Define the type for customStyles
+type CustomStyles = {
+    textPrimary: string;
+    inputBackground: string;
+    buttonBackground: string;
+    accent: string;
+    danger: string;
+    border: string;
+    borderRadius?: string;
+};
+
+// Create a context for customStyles
+const CustomStylesContext = createContext<CustomStyles | undefined>(undefined);
+
+const InputOTP = React.forwardRef<
```

Comment:
The InputOTP file just contains boilerplate components and the EmailOTPInput component uses the InputOTP file. I think it should stay in common only because it could in theory be used for something outside of /auth

Reply:
Not sure what other use cases exist for an OTP outside of auth